src/main/java/com/pracownia/vanet/model/devices/Device.java

```diff
@@ -22,6 +20,8 @@
     protected int id;
     protected Point currentLocation = new Point();
     protected double range;
+    protected UUID privateId = UUID.randomUUID();
```

Comment:
Czemu privateId a nie można skorzystć z globalnie unikatowego id?

Reply:
Tak działa SIN. W założeniach SIN w realnym świecie ID ma być prywatne, aby szkodliwy węzeł nie mógł go skopiować i to właśnie miało tu być odwzorowane.